server/routes/visitJourney/additionalSupport.test.ts

```diff
@@ -20,6 +20,7 @@ const availableSupportTypes = TestData.supportTypes()
 beforeEach(() => {
   flashData = { errors: [], formValues: [] }
   flashProvider.mockImplementation(key => {
+    // @ts-ignore
```

Comment:
You can remove this `@ts-ignore` by changing the line above to:
```typescript
flashProvider.mockImplementation((key: 'errors' | 'formValues') => {
```
So that the index into the  `flashData` object is explicitly typed. Similar fix for most of the other files I think.

(As this is repeated across lots of these test files, we might be able to do something cleverer in future... 🧐)